File: VB/MvcDashboardOverrideOnException/Controllers/DefaultController.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Net
Imports System.Web
Imports System.Web.Mvc
Imports DevExpress.DashboardCommon

Namespace MvcDashboardOverrideOnException

	Public Class DefaultController
		Inherits Controller
		Public Function Index() As ActionResult
			Return View()
		End Function
	End Class
End Namespace

File: CS/MvcCustomTextForInternalDashboardErrors/Controllers/DefaultController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net;
using System.Web;
using System.Web.Mvc;
using DevExpress.DashboardCommon;

namespace MvcCustomTextForInternalDashboardErrors {

    public class DefaultController : Controller
    {
        public ActionResult Index()
        {
            return View();
        }
    }
}